webhooks/charge_invoices.go

```diff
@@ -24,22 +24,31 @@ type ChargeInvoiceNotification struct {
 	Invoice ChargeInvoice `xml:"invoice"`
 }
 
+// https://dev.recurly.com/page/webhooks#section-charge-invoice-schema
```

Comment:
Comment is really helpful but needs to be moved down one line -- first line of the godoc comment should be the `ChargeInvoice` line

Reply:
done